hilti/include/rt/types/vector.h

```diff
@@ -251,15 +404,56 @@ inline std::ostream& operator<<(std::ostream& out, const vector::Empty& x) {
 }
 
 template<typename T, typename Allocator>
-inline std::ostream& operator<<(std::ostream& out, const vector::SafeIterator<T, Allocator>& x) {
```

Comment:
`fmt()` uses the `operator<<` for string formatting, so unless I'm missing something, we should keep them.

Reply:
I missed `fmt`, added them.